cmd/flow-collector/main.go

```diff
@@ -176,6 +237,7 @@ func main() {
 
 		flowRecordTtl = siteConfig.Spec.FlowCollector.FlowRecordTtl
 		enableConsole = siteConfig.Spec.EnableConsole
+		authMode = siteConfig.Spec.AuthMode
```

Comment:
I'm uncertain whether placing this statement here is appropriate. It involves an if/else structure, and within the else block, there is Podman-related code. @fgiorgetti

Reply:
You can add the following block of code into the else part (podman related) and it will get the correct auth mode:

```
		authMode = types.ConsoleAuthModeUnsecured
		if flowUsers != "" {
			authMode = types.ConsoleAuthModeInternal
		}
```